README.md

```diff
@@ -79,16 +81,15 @@ $ref2 = new Reference($obj, function () {});
 var_dump($ref1 == $ref2); // bool(true)
 ```
 
-Strict comparison works as always:
+Strict comparison should still work as expected:
```

Comment:
Both cases are expected =). I originally tried to point that I overloaded compare object handler, so now loose comparison counts hidden from props referent object, but not notifier. Strict comparison is a core php feature and afaik it compare object ce and couldn't be overloaded.